pom.xml

```diff
@@ -199,14 +205,51 @@
                   </excludes>
                 </configuration>
               </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>test-with-sanitizer</id>
+      <activation>
+        <property>
+          <name>USE_SANITIZER</name>
+          <value>ON</value>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>default-test</id>
+                <goals>
+                  <goal>test</goal>
+                </goals>
+                <configuration>
+                  <groups>!noSanitizer</groups>
+                  <jvm>${project.basedir}/build/sanitizer-java/bin/java</jvm>
+                </configuration>
+              </execution>
+              <execution>
+                <!-- Some tests (e.g. error cases) are not suitable to run with sanitizer, so run them separately here -->
+                <id>sanitizer-excluded-cases-test</id>
+                <goals>
+                  <goal>test</goal>
+                </goals>
+                <configuration>
+                  <groups>noSanitizer</groups>
+                </configuration>
+              </execution>
               <execution>
                 <id>non-empty-null-test</id>
                 <goals>
                   <goal>test</goal>
                 </goals>
                 <configuration>
-                  <argLine>-da:ai.rapids.cudf.AssertEmptyNulls</argLine>
-                  <test>ColumnViewNonEmptyNullsTest</test>
```

Comment:
Why are these settings removed -- are we still running this test anywhere?

Reply:
IIUC, this is a duplicate of the definition @ [line554](https://github.com/NVIDIA/spark-rapids-jni/blob/branch-23.10/pom.xml#L554) .